etl/steps/grapher/who/2022-09-30/ghe.py

```diff
@@ -63,3 +66,12 @@ def select_subset_causes(table: pd.DataFrame) -> pd.DataFrame:
     table = table[(table["sex"] == "Both sexes") & (table["age_group"] == "ALLAges")]
 
     return table
+
+
+def add_global_totals(table: pd.DataFrame) -> pd.DataFrame:
+    glob_total = table.groupby(["year", "cause"])[["daly_count", "death_count"]].sum().reset_index()
+    glob_total["country"] = "World"
+    glob_total["age_group"] = "ALLAges"
+    glob_total["sex"] = "Both sexes"
```

Comment:
It feels a bit inconsistent to have "ALLAges" and "Both sexes", but I suppose this choice is taken elsewhere.